autoload/sj/r.vim

```diff
@@ -0,0 +1,190 @@
+" Only real syntax that's interesting is cParen and cConditional
+let s:skip = sj#SkipSyntax(['rComment'])
+
+" function! sj#r#SplitFuncall()
+"
+" Split the R function call if the cursor lies within the arguments of a
+" function call
+"
+function! sj#r#SplitFuncall()
+  if !s:IsValidSelection("va(")
+    return 0
+  endif
+
+  call sj#PushCursor()
+  let items = s:ParseJsonFromMotion("va(\<esc>vi(")
+  let items = map(items, {k, v -> v . (k+1 < len(items) ? "," : "")})
+
+  if g:r_indent_align_args && len(items)
```

Comment:
This works, but I'd consider doing something like this at the top of the function:

``` vim
let align_args = get(g:, 'r_indent_align_args', 1)
```

And then use `align_args`, that way if the R support files change, or if somebody doesn't have the built-in R support (old version?), this code could continue working. I would document this, though.

Reply:
Thanks - I hadn't considered this. I'll add this in.